### ISSUE TITLE
screen doesn't work after jailbreaking

### ISSUE THREAD

**Issue by kitiki1222** [opa334/Dopamine#495]
the screen doesn't work after jailbreaking and then I have do a hard reset
I am on iOS 15.7.2
its an iPhone 7
Dopamine 2.0.9
so pls help

**Comment by opa334** [comment]
Known issue with third party screens, unfortunately this is a stock bug and happens after a userspace reboot even if the jailbreak is not active, there is nothing I can do about it.

**Comment by kitiki1222** [comment]
can I do something about it?

**Comment by eglacias** [comment]
I just finished a 20 hour long project future restoring an 8+ to iOS 16.6.1.
I was able to up date without losing my data, open, troll store from the task manager, and install Dopamine and get everything back working with all my tweaks.

This is the same model that other people have had screen replacements causing the touch response to stop working, etc.

this device has also had a screen replacement, not from the Apple store but from a reputable third-party dealer and I have no problems with the screen.

Therefore, I suggest anyone with this problem go get their screen replaced at an Apple Store, or even just another store with a different screen might solve your issue
